src/main/java/math/Fraction.java

```diff
@@ -178,9 +178,10 @@ public Fraction multiply(Fraction fraction) {
      *
      * @return a fraction that represents the reciprocal of this fraction
      */
-    public Fraction reciprocal() {
-        // TODO (N1SBRH)
-        return null;
+    public Fraction reciprocal() throws ArithmeticException {
+        if (this.numerator == 0)
+            throw new ArithmeticException("Reciprocal doesnt exist.");
```

Comment:
Note that this is unnecessary, an `ArithmeticException` will be thrown by the constructor.